RuntimeException: Unable to resume activity (in UnifiedMapActivity)

We have a couple of reports on support with this error message:

```
 java.lang.RuntimeException: Unable to resume activity {cgeo.geocaching/cgeo.geocaching.unifiedmap.UnifiedMapActivity}: android.util.SuperNotCalledException: Activity {cgeo.geocaching/cgeo.geocaching.unifiedmap.UnifiedMapActivity} did not call through to super.onResume()
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5234)
at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5267)
at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:57)
at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:190)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:101)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2604)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:222)
at android.os.Looper.loop(Looper.java:314)
at android.app.ActivityThread.main(ActivityThread.java:8857)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:565)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
Caused by: android.util.SuperNotCalledException: Activity {cgeo.geocaching/cgeo.geocaching.unifiedmap.UnifiedMapActivity} did not call through to super.onResume()
at android.app.Activity.performResume(Activity.java:9080)
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5222)
... 13 more
```

support tickets 640371, 344545 and 413764, all for a recent nightly version (2025.12.30-NB being the oldest one)

Additional info from one user: "I added a waypoint to a cache in dd.ddddd format. Unable to reproduce."

--- a/main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java
+++ b/main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java
@@ -1457,12 +1457,12 @@ public class UnifiedMapActivity extends AbstractNavigationBarMapActivity impleme
 
     @Override
     protected void onResume() {
+        super.onResume();
         if (mapFragment == null) {
             recreate(); // restart with a fresh MapView
             return; // prevent further execution on the old activity instance
         }
 
-        super.onResume();
         reloadCachesAndWaypoints();
         MapUtils.updateFilterBar(this, viewModel.mapType.filterContext);
 

